docs/ota-client-guide/modules/ROOT/nav.adoc

```diff
@@ -62,6 +62,11 @@ ifndef::env-github[:pageroot:]
 ** xref:{pageroot}hsm-provisioning-example.adoc[Generate a device certificate using an HSM]
 ** xref:{pageroot}enable-device-cred-provisioning.adoc[Enable and install device certificates]
 
+* xref:{pageroot}test-root-cert.adoc[Testing with a root certificate]
+** xref:{pageroot}provide-testroot-cert.adoc[Register your test root certificate]
+** xref:{pageroot}generatetest-devicecert.adoc[Generate a test device certificate]
+** xref:{pageroot}enable-device-cred-provtest.adoc[Enable and install test device certificates]
```

Comment:
I didn't realize all these pages were hidden until now!

Reply:
@patrickvacek I realized they were hidden after going through the developer portal doc. I'm not sure if all the information is still up to date but I tried to make these easier to find on the doc portal.